README.md

```diff
@@ -168,9 +195,73 @@ $> export no_proxy=localhost,127.0.0.0/8,::1
 $> export HTTP_PROXY=$http_proxy;export HTTPS_PROXY=$https_proxy;export NO_PROXY=$no_proxy;
 ```
 
+### Windows
+
+Open `x64 Native Tools Command Prompt for VS2019` and run the following commands:
+
+``` batch
+> git config --global core.eol lf
+> git config --global core.autocrlf input
+> git clone https://github.com/dynawo/dynawo.git dynawo
+> cd dynawo
+> cmake -S dynawo/3rdParty -B b-3-p -DCMAKE_INSTALL_PREFIX=../d-3-p -DOPENMODELICA_INSTALL=../OpenModelica/Install -DOPENMODELICA_SRC=../OpenModelica/Source -DOMDEV_HOME=../OMDev -G "NMake Makefiles"
+> cmake --build b-3-p
+> cmake -S dynawo -B b -DCMAKE_INSTALL_PREFIX=../d-i -DDYNAWO_HOME=.. -DINSTALL_OPENMODELICA=../../OpenModelica/Install -DDYNAWO_THIRD_PARTY_DIR=../../d-3-p -G "NMake Makefiles
+> cmake --build b --target install
+> cmake --build b --target models
+> cmake --build b --target solvers
+```
+
+**Warning** We try to limit as far as possible the name of the build and install folders (for example d-i instead of dynawo-install or b-3-p for build-3rd-parties) because of Windows limitation of length of path for folders. We know it causes problems and the only solution is to install Dyna&omega;o in a shorter length directory path.
+
+**Warning** Only the build directories (b and b-3-p) can be located in the `dynawo` folder, the install (d-i and d-3-p), OMDev and OpenModelica folders should be located outside to avoid problems with CMake.
+
+### MacOS
+
+The difficult part of building Dyna&omega;o on MacOS is the compilation of OpenModelica. We propose two solutions, one with pre-built binaries for OpenModelica and one with building OpenModelica.
+
+#### With pre-built OpenModelica
+
+We provide on Github MacOS binaries of Dyna&omega;o, part of those are OpenModelica Compiler binaries built for Dynawo. You can download those binaries and use them to build Dyna&omega;o. You can launch the following commands in a terminal:
```

Comment:
Use the following commands in a terminal to download those binaries and use them to build Dyna&omega;o

Reply:
done